### PR TITLE
Add documentation for bkk integration

### PR DESCRIPTION
**Pull request by gabor-simon**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:
- This PR fixes or closes issue: -

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/bkk.markdown [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: "Budapest Public Transport"
+description: "Instructions on how to integrate data from the BKK Futar API into Home Assistant."
+logo: bkk.png
+ha_category:
+  - Transport
+ha_release: 0.49
```
Comment (klaasnicolaas) [inline source/_integrations/bkk.markdown]:
```suggestion
ha_release: 0.105
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/bkk.markdown [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+```
+
+{% configuration %}
+stop:
+  description: id of the monitored stop. For getting a stop id, see description below.
+  required: true
+  type: string
+grouped:
+  description: show routes grouped on UI
+  required: no
```
Comment (klaasnicolaas) [inline source/_integrations/bkk.markdown]:
```suggestion
  required: false
```

File: source/_integrations/bkk.markdown [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  type: boolean
+  default: false
+only_departures:
+  description: show only departures
+  required: no
```
Comment (klaasnicolaas) [inline source/_integrations/bkk.markdown]:
```suggestion
  required: false
```

File: source/_integrations/bkk.markdown [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  type: boolean
+  default: true
+minutes_after:
+  description: minutes from current time to query departures for
+  required: no
```
Comment (klaasnicolaas) [inline source/_integrations/bkk.markdown]:
```suggestion
  required: false
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/bkk.markdown (hunk 1 above)
Comment (gabor-simon) [reply source/_integrations/bkk.markdown]: thanks, done!

File: source/_integrations/bkk.markdown (hunk 4 above)
Comment (gabor-simon) [reply source/_integrations/bkk.markdown]: Done

File: source/_integrations/bkk.markdown (hunk 3 above)
Comment (gabor-simon) [reply source/_integrations/bkk.markdown]: Done

File: source/_integrations/bkk.markdown (hunk 2 above)
Comment (gabor-simon) [reply source/_integrations/bkk.markdown]: Done

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by frenck** [comment]
Closing PR, since the parent PR has been closed.

**Pull request closed without merging**